src/cursor/cur_backup.c

```diff
@@ -299,6 +300,11 @@ __wt_curbackup_open(WT_SESSION_IMPL *session, const char *uri, WT_CURSOR *other,
         F_SET(cb, WT_CURBACKUP_QUERYID);
     }
 
+    /* Special backup cursor for export operation. */
+    if (strcmp(uri, "backup:export") == 0) {
```

Comment:
I would prefer that we pass a boolean variable into `backup_start` instead of creating a flag here